src/storage/src/hummock/sstable/delete_range_aggregator.rs

```diff
@@ -124,8 +156,19 @@ impl RangeTombstonesCollector {
                 continue;
             }
 
-            if self.gc_delete_keys && tombstone.sequence <= self.watermark {
-                continue;
+            if tombstone.sequence <= self.watermark {
+                if self.gc_delete_keys {
```

Comment:
why?

Reply:
We can drop this range-tombstone when it reach the bottommost level just like other normal delete-tombstone.